doc/configuration/database-backends-configuration.md

```diff
@@ -47,12 +55,13 @@ Persistent Data:
 
 * ElasticSearch - Only for MAM (Message Archive Management).
 
+* Mnesia - some backends support Mnesia to store data, but it is not recommended.
+
```

Comment:
This pretty much duplicates the warning from above. We could move such warning actually exactly to this section.